configure.ac

```diff
@@ -338,6 +338,7 @@ AC_CONFIG_FILES([
 	tests/zfs-tests/tests/functional/inheritance/Makefile
 	tests/zfs-tests/tests/functional/inuse/Makefile
 	tests/zfs-tests/tests/functional/io/Makefile
+	tests/zfs-tests/tests/functional/l2arc_arcstats/Makefile
```

Comment:
I think it would be fine to add these to the existing `tests/zfs-tests/tests/functional/arc/` directory.  Or even a new `l2arc` directory which is a little broader in scope.